guides/request-validation-java/test.yml

```diff
@@ -0,0 +1 @@
+testable: true
```

Comment:
This is one is necessary, but the one in `/guides` is not, as the default behavior when there is no test.yml is not to test.